spec/system/admin/products_spec.rb

```diff
@@ -347,13 +347,28 @@
       end
 
       it "editing a product" do
+        login_as_admin
         visit spree.edit_admin_product_path product
 
         fill_in_trix_editor 'product_description', with: 'A description...'
+
         click_button 'Update'
+
         expect(flash_message).to eq('Product "a product" has been successfully updated!')
         product.reload
         expect(product.description).to eq("<div>A description...</div>")
+
+        # Product preview
+        click_link 'Preview'
+
+        within "#product-preview-modal" do
+          expect(page).to have_content("Product preview")
+          expect(page).to have_selector("h3 a span", text: "a product")
+
```

Comment:
I would usually agree to avoid doubling up tests in feature specs, but I think it would be helpful to test that the tab is working in this context (and it certainly won't add much runtime to the test).
Perhaps the test could be a shared_example?